etc/kayobe/pulp.yml

```diff
@@ -477,6 +477,7 @@ stackhpc_pulp_images_kolla:
   - neutron-dhcp-agent
   - neutron-l3-agent
   - neutron-metadata-agent
+  - neutron-mlnx-agent
```

Comment:
I was expecting the eswitchd image in here?

Do we need to install networking-mlnx into the nova-compute image as well, so os-vif can use the ebrctl?
https://opendev.org/x/networking-mlnx/src/commit/0a69b9715505f21fe7cca8c408c42b618d2e72d7/setup.cfg#L55

Reply:
ah, sorry, there isn't one, even better!